Makefile

```diff
@@ -35,6 +35,7 @@ GIT_COMMIT = $(shell git rev-parse --short HEAD)
 AUTHOR ?= $(shell git config user.email || echo $$USER)
 
 ENABLE_METRICS ?= true
+BUILD_TAGS ?= gowaku_no_rln
```

Comment:
Hi @siphiuel : Can you please elaborate on what this build flag does and why we need it ? :)

Reply:
Good question! The short answer is that i don't know, a bit more detailed one is: @richard-ramos told me i should add it in order to make builds work :)

Reply:
RLN is no longer experimental in waku, however it is currently unsupported in mobile. To avoid compile issues, this tag was created so RLN and related code are not included in the vendor folder

Reply:
Thanks for explaining @siphiuel @richard-ramos